Command/DumpCommand.php

```diff
@@ -81,6 +87,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this
             ->getContainer()
             ->get('bazinga.jstranslation.translation_dumper')
-            ->dump($this->targetPath, $formats, $merge);
+            ->dump($this->targetPath, $this->getTranslationsPath($input->getOption('path')), $formats, $merge);
+    }
+
+    /**
+     * Get translations path from the router defined or use the default one.
+     */
+    private function getTranslationsPath($path)
+    {
+        $translationsPathOverriden = empty($path) ? 'translations' : $path;
+        $route = $this->getContainer()->get('router')->getRouteCollection()->get('bazinga_jstranslation_js');
```

Comment:
nope, as this is what is adding it.

You should use the router only when the CLI option is not used at all, and make the option pass the full pattern you need.